### ISSUE TITLE
Update Google Account used to access maps API

### ISSUE THREAD

**Issue by dshaykewich** [BCcampus/eypd#566]
### Description 

Currently, Brad's personal Google account is being used to access Google APIs for embedding maps into the site.  There is now a BCcampus account, which should be used instead.

**Expected behaviour** 

Use the "devops@bcccampus.ca" google account to access google maps apis.

**Actual behaviour** 

Using Brad's personal account.

**Comment by bdolor** [comment]
this is complete...will likely take a few minutes to propagate

**Comment by mandily-p** [comment]
Looks like there's something up with the css on the minus and plus size icons in the lower right.
![screen shot 2018-11-27 at 10 56 38 am](https://user-images.githubusercontent.com/20425828/49104655-56e67d80-f233-11e8-9842-94a16982c53c.png)

**Comment by bdolor** [comment]
if the icon display is connected to the scope of work outlined in the ticket it can be addressed here, otherwise it may warrant opening up a new ticket.

**Comment by mandily-p** [comment]
Good call. I assumed it was related. It's not. I will open up a new ticket.

**Comment by mandily-p** [comment]
Otherwise, the maps are loading so this looks 👍 to me.
